tests/memgraph/test_query_builder.py

```diff
@@ -428,6 +429,63 @@ def test_orderby_desc(memgraph):
     mock.assert_called_with(expected_query)
 
 
+def test_order_by_asc(memgraph):
+    query_builder = QueryBuilder().match().node(variable="n").return_().order_by(("n.id", "ASC"))
+    expected_query = " MATCH (n) RETURN * ORDER BY n.id ASC "
+
+    with patch.object(Memgraph, "execute_and_fetch", return_value=None) as mock:
+        query_builder.execute()
+
+    mock.assert_called_with(expected_query)
+
+
+def test_order_by_wrong_ordering(memgraph):
+    with pytest.raises(GQLAlchemyMissingOrdering):
+        QueryBuilder().match().node(variable="n").return_().order_by(("n.id", "DESCE")).execute()
```

Comment:
It is not clear to me why `GQLAlchemyMissingOrdering` error is being raised only if I add `execute() `at the end of the query builder. I did [a similar thing in `where()`](https://github.com/memgraph/gqlalchemy/pull/111/files#diff-b58f85388321491b6930511b2eae02b960636ea7c5b2d2ab20b465dc735471f6R344) clause and `execute()` wasn't necessary.

Reply:
Because we are doing checks at the time of the execution and not in creation time.